### ISSUE TITLE
uMatrix blocks sites with no way to unblock

### ISSUE THREAD

**Issue by kstenerud** [gorhill/uMatrix#640]
Steps to reproduce:

- Go to google.com
- Type in a search term
- Click one of the "ad" results
- uMatrix blocks it
- Open uMatrix and enable everything
- Reload page

Result: page still refuses to load

Screenshot:
![screen shot 2016-10-09 at 2 18 06 am](https://cloud.githubusercontent.com/assets/245857/19219488/44981682-8dc9-11e6-8920-b460400f37a9.png)

On a side note, it would be nice to have a "Load anyway" button, rather than having to do a ton of clicks inside the matrix to get the page to (maybe) unblock.

**Comment by gorhill** [comment]
> Open uMatrix and enable everything

Why? Just remove the blacklist rule for the hostname which is reported as blocked (truncated in your screenshot, can't tell) then force a reload.
